packages/viz/src/lib/sources/CARTOSource.ts

```diff
@@ -13,6 +13,9 @@ export interface SourceOptions {
 const defaultMapOptions: MapOptions = {
   vectorExtent: 2048,
   vectorSimplifyExtent: 2048,
+  bufferSize: {
+    mvt: 10
```

Comment:
@manmorjim @alasarr Is this default valid for any type of geometry / style? why 10?

Reply:
Yes. I added 10 to support because before at the map instantiation we don't know which type of geometry. I'd keep like that and talk if it makes sense to include at maps API a buffer based on the geom type